src/Dev/SapphireTest.php

```diff
@@ -436,7 +438,9 @@ public function setFixtureFactory(FixtureFactory $factory)
      */
     protected function idFromFixture($className, $identifier)
     {
-        $id = $this->getFixtureFactory()->getId($className, $identifier);
+        /** @var FixtureTestState $state */
+        $state = static::$state->getStateByName('fixtures');
```

Comment:
I wonder if we should just un-deprecate getFixtureFactory(). This logic is repeated so much.

@dhensby what do you think?

Reply:
The problem is that it should be a static method... maybe we just need to change it to static?

Reply:
I can un-deprecate the method and make it static if we're happy to do that

Reply:
I don't think making it static is a good idea until 5 because (although very unlikely) any extension that overrides this will break if you make it static

Reply:
What would you guys like me to do from here?